schema_based_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+-- Create the patient table
+CREATE TABLE patients ( 
+    id integer primary key NOT NULL,
+    name varchar(100), 
+    date_of_birth date 
+);
+
+-- Create the medical_histories table
+CREATE TABLE medical_histories ( 
+    id INTEGER GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    admitted_at timestamp NOT NULL, 
+    patient_id INT REFERENCES patients(id), 
+    status varchar(50)
+);
+CREATE INDEX index_patient_id ON medical_histoeries (patient_id);
+
+-- Create the invoices table
+CREATE TABLE invoices ( 
+    id INTEGER GENERATED ALWAYS AS IDENTITY PRIMARY KEY, 
+    total_amount decimal NOT NULL,
+    generated_at timestamp NOT NULL, 
+    payed_at TIMESTAMP,
+    medical_history_id INTEGER REFERENCES medical_histories (id)
+);
+CREATE INDEX index_medical_history_id ON invoices (medical_history__id);
+
+-- Create the treatments table
+CREATE TABLE treatments (
+    id INTEGER GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    type varchar(20) NOT NULL,
+    name varchar(50) NOT NULL
+);
+
+-- create invoice items table
+CREATE TABLE invoice_items (
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    unit_price DECIMAL NOT NULL,
+    quantity INT NOT NULL,
```

Comment:
- [ ] You are currently missing `total_price` from the invoice_items table. Kindly look at the ERD and add it.

Reply:
@Ghiftee Done.